packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/schema/CachedClassKeyMap.kt

```diff
@@ -58,8 +60,10 @@ public interface PropertyMetadata {
     public val type: PropertyType
     public val isNullable: Boolean
     public val isPrimaryKey: Boolean
-    public val acccessor: KMutableProperty1<BaseRealmObject, Any?>?
-    public val linkTarget: String?
+    public val accessor: KProperty1<BaseRealmObject, Any?>?
+    public val linkTarget: String
```

Comment:
no longer nullable?

Reply:
Yes, it is no longer nullable, it always has a value.